packages/x-bundle/src/validators/ObjectId.validator.ts

```diff
@@ -18,7 +18,7 @@ export class ObjectIdSchema extends yup.BaseSchema<any> {
 
   protected _typeCheck(_value: any): _value is NonNullable<any> {
     try {
-      return ObjectId.isValid(_value);
+      return ObjectId.isValid(_value) || ObjectId.isValid(_value.toString());
```

Comment:
I want to understand this, why is .toString() needed?

Reply:
Seems that the object id validate only ids as strings not as objectid